source/MaterialXCore/Look.cpp

```diff
@@ -19,7 +19,7 @@ const string Visibility::VISIBILITY_TYPE_ATTRIBUTE = "vistype";
 const string Visibility::VISIBLE_ATTRIBUTE = "visible";
 
 const string LookGroup::LOOKS_ATTRIBUTE = "looks";
-const string LookGroup::ACTIVE_ATTRIBUTE = "active";
+const string LookGroup::ENABLED_ATTRIBUTE = "enabled";
```

Comment:
You can't change this string as it's defined in the spec. If it's changed we need an upgrade path.